prow/genjobs/gen-private-jobs.sh

```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit
```

Comment:
This was better (more readable and obvious)

Reply:
I have a keyboard macro that generates this using *short opt* flags. If you prefer *long opts* format then I will revise.

Reply:
Updated